packages/integrations/preact/src/index.ts

```diff
@@ -60,7 +60,21 @@ export default function ({ include, exclude, compat, devtools }: Options = {}):
 					};
 				}
 
-				viteConfig.plugins = [preactPlugin];
+				viteConfig.plugins = [
+					preactPlugin,
+					// TODO: remove once https://github.com/preactjs/preset-vite/pull/156 is released
```

Comment:
Just released as 2.10.1, thanks for letting us know this was an issue!

Reply:
Thanks for being so reactive!